charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml

```diff
@@ -28,6 +28,8 @@ annotations: {}
 tolerations:
   - effect: NoSchedule
     operator: Exists
+  - effect: NoExecute
+    operator: Exists
```

Comment:
Whats purpose of adding this ?

Reply:
This was from when I was testing I will remove it